jekyll/_cci2/concepts.md

```diff
@@ -78,60 +112,37 @@ An image is a packaged system that has the instructions for creating a running c
      # an Ubuntu version 14.04 image with Docker 17.06.1-ce
      # and docker-compose 1.14.0, follow CircleCI Discuss Announcements
      # for new image releases.
-       image: circleci/classic:201708-01
+       image: ubuntu-1604:201903-01
 #...       
    build3:
      macos: # Specifies a macOS virtual machine with Xcode version 11.3
        xcode: "11.3.0"
 # ...          
  ```
 
+The Primary Container is defined by the first image listed in [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file. This is where commands are executed. The Docker executor spins up a container with a Docker image. The machine executor spins up a complete Ubuntu virtual machine image. See [Choosing an Executor Type]({{ site.baseurl }}/2.0/executor-types/) document for a comparison table and considerations. Subsequent images can be added to spin up Secondary/Service Containers.
 
-## Jobs
-
-Jobs are collections of [steps](#steps). Each job must declare an executor that is either `docker`, `machine`, `windows` or `macos`. `machine` includes a [default image](https://circleci.com/docs/2.0/executor-intro/#machine) if not specified, for `docker` you must [specify an image](https://circleci.com/docs/2.0/executor-intro/#docker) to use for the primary container, for `macos` you must specify an [Xcode version](https://circleci.com/docs/2.0/executor-intro/#macos), and for `windows` you must use the [Windows orb](https://circleci.com/docs/2.0/executor-intro/#windows).
+When using the docker executor and running docker commands, the `setup_remote_docker` key can be used to spin up another docker container in which to run these commands, for added security. For more information see the [Running Docker Commands]({{ site.baseurl }}/2.0/building-docker-images/#accessing-the-remote-docker-environment) guide.
 
-![job illustration]( {{ site.baseurl }}/assets/img/docs/concepts1.png)
-
-### Cache
+#### Steps
 {:.no_toc}
 
-A cache stores a file or directory of files such as dependencies or source code in object storage.
-Each job may contain special steps for caching dependencies from previous jobs to speed up the build.
-
-{% raw %}
+Steps are actions that need to be taken to complete your job. Steps are usually a collection of executable commands. For example, the [`checkout`]({{ site.baseurl }}//2.0/configuration-reference/#checkout) step checks out the source code for a job over SSH. Then, the `run` step executes the `make test` command using a non-login shell by default. Commands can also be defined [outside the job declaration]({{ site.baseurl }}/2.0/configuration-reference/#commands-requires-version-21), making them reusable across your config.
```

Comment:
```suggestion
Steps are actions that need to be taken to complete your job. Steps are usually a collection of executable commands. For example, the [`checkout`]({{ site.baseurl }}//2.0/configuration-reference/#checkout) step, which is a _built-in_ step available across all CircleCI projects, checks out the source code for a job over SSH. Then, the `run` step allows you to run custom commands, such as executing the command `make test`  using a non-login shell by default. Commands can also be defined [outside the job declaration]({{ site.baseurl }}/2.0/configuration-reference/#commands-requires-version-21), making them reusable across your config.
```

Just a suggestion to differentiate between reserved/built-in steps and a user's custom steps. I'm not sure if the "make test" is referring to anything specific, so my suggestion tries to make clear that it is a custom command.

Reply:
Yes this is definitely clearer now, thank you!!